packages/main/test/specs/MultiInput.spec.js

```diff
@@ -418,28 +430,30 @@ describe("Keyboard handling", () => {
 	});
 
 	it("tests if tokenizer is scrolled on keyboard navigation through the tokens", async () => {
```

Comment:
please do not modify existing tests add new if you want to change the tested HTML element